lib/index.js

```diff
@@ -27,6 +27,17 @@ class PWMetrics {
     this.sheets = opts.sheets;
     this.expectations = opts.expectations || false;
     this.tryLighthouseCounter = 0;
+    this.desiredMetrics = [];
+    if (this.flags.desiredMetrics) {
+      this.desiredMetrics = this.flags.desiredMetrics.filter(metric => {
+        if (!Object.values(metrics.ids).includes(metric)) {
+          console.log(getMessageWithPrefix('WARNING', 'UNKOWN_METRIC', metric));
+          return false;
+        }
+        return true;
+      });
+    }
+    this.desiredMetrics = this.desiredMetrics.length ? this.desiredMetrics : ['tti', 'psi', 'ttfmp', 'ttfcp'];
```

Comment:
Because it would send an error to https://github.com/pedro93/pwmetrics/blob/145b489f98636a5d37c7f2e5fe8523deed965f38/lib/index.js#L74 which does not have a catch clause. Suppose the run 4/10 throws, by throwing the error, the remaining runs are not completed:

With the throw Error I had to manually cancel the process (gave it 5m to recover).
![screen shot 2017-03-16 at 09 19 25](https://cloud.githubusercontent.com/assets/3718654/23989243/b191451a-0a29-11e7-88f9-3f5d71423f98.png)